src/main/java/edu/hm/hafner/analysis/parser/SimulinkCheckParser.java

```diff
@@ -48,7 +49,8 @@ public Report parse(final ReaderFactory readerFactory) throws ParsingException {
             }
             var report = new Report();
 
-            var system = systemElement.id();
+            var modelNameElement = systemElement.selectFirst(MODEL_NAME_SELECTOR);
+            var system = modelNameElement.text();
```

Comment:
`modelNameElement ` might  be `null` so this needs to wrapped in an `if` (and the alternative is using the existing value?).

Reply:
Can you also add an assertion in the corresponding test for the new value?